examples/shared-vpc/README.md

```diff
@@ -0,0 +1,35 @@
+# Shared Virtual Private Cloud Networking in Google Cloud
+
+This is a template showcasing the shared VPC feature in Google Cloud.  It features
+four projects:
+- A host project, which owns a VPC
+- Two service projects, each of which owns a VM connected to the VPC
+- A fourth project, which owns a VM not connected to the VPC.
+
+It is based on the diagram in the overview at [https://cloud.google.com/vpc/docs/shared-vpc](https://cloud.google.com/vpc/docs/shared-vpc).
+
+This example creates projects within an organization - to run it, you will need to have an Organization ID.  To get started using Organizations, read the quickstart [here](https://cloud.google.com/resource-manager/docs/quickstart-organizations).  Since it uses organizations, project-specific credentials won't work, and consequently this example is configured to use [application default credentials](https://developers.google.com/identity/protocols/application-default-credentials).  Ensure that the application default credentials have permission to create and manage projects and Shared VPCs (sometimes called 'XPN').
```

Comment:
It is possible to grant organization-level permissions to project-specific credentials such as a service account.

In the UI, select your org in the top dropdown menu, visit Iam & Admin > Iam. Then click the +Add button at the top to add new members. In the members field, you can put a service account email.